diff --git a/lib/middleware.d.ts b/lib/middleware.d.ts
index 012d0a5cc56de688d876ddcaf77df29bca63fa32..8e0436c92896c160f440f9e6dcc10006b7192aa0 100644
--- a/lib/middleware.d.ts
+++ b/lib/middleware.d.ts
@@ -1,5 +1,7 @@
 import type { Parser, Node, TokenType, Position } from 'acorn';
 import type { Options } from 'acorn';
+
+// @ts-ignore
 export declare class AcornParseClass extends Parser {
     options: Options & {
         onComment: any;
diff --git a/lib/types.d.ts b/lib/types.d.ts
index 721fce33207a9442608a863818e761eed4bd4ac4..604ff53731e334effc859416ee1a37737c4ebacf 100644
--- a/lib/types.d.ts
+++ b/lib/types.d.ts
@@ -1,3 +1,4 @@
+// @ts-ignore
 import type { Position, TokContext, TokenType } from 'acorn';
 export declare type Accessibility = 'public' | 'protected' | 'private';
 export declare type VarianceAnnotations = 'in' | 'out';
